### ISSUE TITLE
Fabric_NavTree : State of menu/sub-menu item is not narrated by screen reader

### ISSUE THREAD

**Issue by v-vasud** [OfficeDev/office-ui-fabric-react#8310]
### Environment Information 
OS: Windows 10
Version: 1809(OS Build 18298.1000)
Browser: Microsoft Edge (44.18298.1000.0)
Screen Reader: Narrator

#### User Experience:
Users who rely on screen reader are getting affected as the state(Selected/Unselected) of the menu items is not announced by the narrator.

#### Reproduction Steps:
1) Turn on Narrator
2) Open https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/nav#Implementation URL in Edge browser
3) "Nav Implementation" page will be displayed
4) Navigate to menu/sub-menu items and observe whether screen reader is narrating the state of menu/sub-menu item.

#### Actual behavior:

Narrator is not reading the state (Selected/Expanded/Collapsed) of menu items.

#### Expected behavior:

Narrator should read the state (Selected/Expanded/Collapsed) of menu items.

#### Priorities and help requested:

Requested priority: (High) 
 
Products/sites affected: M365 Security and compliance

Impacted bugs due to this fabric issue
[Bug 1](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fo365exchange.visualstudio.com%2FIP%2520Engineering%2F_workitems%2Fedit%2F962028&data=02%7C01%7Cv-bhachu%40microsoft.com%7C280db32e182f484bf29308d6a84821ed%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636881425098148848&sdata=X4jFi41yALZ4jb%2F%2FKDEZRq7MbgKSoo9OVh7jY3PD%2FEs%3D&reserved=0) : Narrator is not reading the Link Current state (Selected) at left navigation menu region
[Bug 2](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fo365exchange.visualstudio.com%2FIP%2520Engineering%2F_workitems%2Fedit%2F963061&data=02%7C01%7Cv-bhachu%40microsoft.com%7C280db32e182f484bf29308d6a84821ed%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636881425098178877&sdata=9szGs26f%2FT178JdhufB0zINiaGHn5rQiUcTyh5s46SA%3D&reserved=0) : Narrator is not announcing the expanded/collapsed state for the hamburger menu.
[Bug 3](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fo365exchange.visualstudio.com%2FIP%2520Engineering%2F_workitems%2Fedit%2F963065&data=02%7C01%7Cv-bhachu%40microsoft.com%7C280db32e182f484bf29308d6a84821ed%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636881425098198890&sdata=WMFC3H8gbigQQmpsVH5Z4wPaDCeHEP46jMEx6onqTX4%3D&reserved=0) : Narrator is not narrating descriptive information about menu items.

**Comment by Jahnp** [comment]
This issue--and a bunch of the subsequent issues targeting https://security.microsoft.com--really don't seem like issues for Fabric controls at all; they're scenario-specific. The control in this issue looks like a custom Angular directive and doesn't relate to Fabric.

@v-vasud can you please verify if these issues should actually target Fabric or not?

**Comment by KevinTCoughlin** [comment]
I inspected the DOM, those controls aren't Fabric. They are angular based.

**Comment by KevinTCoughlin** [comment]
@v-vasud, some follow-up questions:

**Bug 1 : Narrator is not reading the Link Current state (Selected) at left navigation menu region**

If this bug is in reference to the selected link not being read aloud by Narrator, I believe I can repro this with basic Nav examples. See Narrator output below:

```shell
Breadcrumb, link,
CommandBar, link,
Space
```

**Bug 2 : Narrator is not announcing the expanded/collapsed state for the hamburger menu.**

What hamburger menu? To my knowledge Fabric does not ship a "hamburger menu".

@v-vasud, by "hamburger menu" are you referring to the following:

**Expanded**

![image](https://user-images.githubusercontent.com/706967/54397573-30c6db00-4674-11e9-8918-e25fdd1cd14e.png)

**Collapsed**

![image](https://user-images.githubusercontent.com/706967/54397568-2ad0fa00-4674-11e9-80fd-8b18939ac9db.png)


**Bug 3 : Narrator is not narrating descriptive information about menu items.**

@v-vasud can you clarify what the expected output for Bug 3 is? Specifically what you mean by "descriptive information".

Do you expect _all_ links to read their _selected_ state or only those which are _selected_.

Not that in a Nav grouping only one element can be selected at a time.

**Comment by KevinTCoughlin** [comment]
After speaking with @v-vasud on Friday, we came to the conclusion that the `Nav` with accessibility issues was related to the wrapper component within the Dashboards package. Adding the appropriate folks to triage cc: @Raghurk.

If this turns out to be OUFR `Nav` feel free to re-engage me.

**Comment by ecraig12345** [comment]
Closing this because the dashboard package has moved to a new location.
